pymc3/distributions/__init__.py

```diff
@@ -141,7 +143,7 @@ def change_rv_size(
 
 def rv_log_likelihood_args(
     rv_var: TensorVariable,
-    transformed: Optional[bool] = True,
+    return_observations: Optional[bool] = True,
```

Comment:
Would it be a good practice to require `return_observations` to be a kwarg?
The rule of thumb I know if is that one should do that if the passed value (True/False) does not imply the function.

Reply:
You mean recast `return_observations` as the "don't return observations"?

Reply:
No, just to insert a `, *,` such that writing `return_observations=` is required

Reply:
Oh, I see; yeah, we can do that.